elm.json

```diff
@@ -9,9 +9,11 @@
             "elm/browser": "1.0.2",
             "elm/core": "1.0.4",
             "elm/html": "1.0.0",
+            "wernerdegroot/listzipper": "4.0.0",
             "elm-community/list-extra": "8.2.2",
             "elm-community/maybe-extra": "5.1.0",
-            "mdgriffith/elm-ui": "1.1.5"
+            "mdgriffith/elm-ui": "1.1.5",
+            "wernerdegroot/listzipper": "4.0.0"
```

Comment:
Have you manually edited `elm.json`?
`elm install wernerdegroot/listzipper` wouldn't have added it twice.